README.md

```diff
@@ -249,7 +327,6 @@ In order to test the entire prototype, execute:
 # 6: Documentation
 * [TMC Prototype (Stage II) Design Document](https://docs.google.com/document/d/1DaLCQZ79yMlGYm2GPE03KpQ80kzjb1f98bWhgV_2Rpo/edit?usp=sharing)
 * [ReadTheDocs](https://developer.skatelescope.org/projects/tmc-prototype/en/master/)
-* [OSO-UI / TMC PI2 meeting notes](https://docs.google.com/document/d/1sEEXwSMJatzFlP1iIw1d4ZrtEgvYvOS-_FPBmKd1i9A/edit)
-* [TMC Stage I Prototype Demo](https://replay.skatelescope.org/replay/showRecordingExternal.html?key=3Na630an0coGgdK)
```

Comment:
I am in the opinion to keep this old demo link. As the UI is Taurus one may refer it later. Also, it gives a good idea of TMC role for an observation.

Reply:
Okay. It will be kept for reference.

Reply:
Commit ID: [4fe0b933c9bfe4e5c66c31e0dfae0261bf485831]